examples/evaluate_lightning_text_classification.py

```diff
@@ -0,0 +1,36 @@
+import pprint
+from pathlib import Path
+
+from embeddings.defaults import RESULTS_PATH
+from embeddings.pipeline.lightning_text_classification import TextClassificationPipeline
+
+root: Path = RESULTS_PATH.joinpath("lightning_sequence_classification")
+root.mkdir(parents=True, exist_ok=True)
+
+# pipeline = LightningClassificationPipeline(
+#     embedding_name="allegro/herbert-base-cased",
+#     dataset_name="clarin-pl/cst-wikinews",
+#     input_column_name=["sentence_1", "sentence_2"],
+#     target_column_name="label",
+#     task_train_kwargs={"max_epochs": 1, "gpus": 1},
+#     task_model_kwargs={"pool_strategy": "cls", "learning_rate": 5e-4}
+# )
+
+pipeline = TextClassificationPipeline(
```

Comment:
is it `LightningClassificationPipeline`? It would be good to differentiate it from flair-based pipelines. We can name this pipeline as `HuggingFaceClassificationPipeline` and rename actual `HuggingFaceClassificationPipeline` to `FlaiClassificationPipeline`, wdyt?

Reply:
I agree. It should be renamed.